src/bacnet/basic/object/av.c

```diff
@@ -904,18 +904,24 @@ bool Analog_Value_Write_Property(BACNET_WRITE_PROPERTY_DATA *wp_data)
                 /* Command priority 6 is reserved for use by Minimum On/Off
                    algorithm and may not be used for other purposes in any
                    object. */
-                if (Analog_Value_Present_Value_Set(wp_data->object_instance,
-                        value.type.Real, wp_data->priority)) {
-                    status = true;
-                } else if (wp_data->priority == 6) {
-                    /* Command priority 6 is reserved for use by Minimum On/Off
-                       algorithm and may not be used for other purposes in any
-                       object. */
-                    wp_data->error_class = ERROR_CLASS_PROPERTY;
-                    wp_data->error_code = ERROR_CODE_WRITE_ACCESS_DENIED;
+                if (CurrentAV->Out_Of_Service == true) {
+                    if (Analog_Value_Present_Value_Set(wp_data->object_instance,
+                                value.type.Real, wp_data->priority)) {
+                        status = true;
+                    } else if (wp_data->priority == 6) {
```

Comment:
I know this was already used in the code previously, but as an improvement, could we maybe set magic number to 
a sensible MACRO value instead?

Reply:
it is in the original code, hence we try not to modify it for the sake of future merges